op-deployer/pkg/deployer/clean/cache.go

```diff
@@ -0,0 +1,44 @@
+package clean
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/urfave/cli/v2"
+)
+
+func CacheCLI(cliCtx *cli.Context) error {
+	logCfg := oplog.ReadCLIConfig(cliCtx)
+	l := oplog.NewLogger(oplog.AppOut(cliCtx), logCfg)
+	oplog.SetGlobalLogHandler(l.Handler())
+
+	return CleanCache(l)
+}
+
+func CleanCache(l log.Logger) error {
+	if err := os.RemoveAll("/tmp/op-deployer-cache"); err != nil && !os.IsNotExist(err) {
+		return fmt.Errorf("failed to remove cache directory: %w", err)
+	}
+
+	varFolders := "/var/folders"
```

Comment:
Why do we have to do this? This seems OS X only, and could be dangerous on other systems.